crates/nu-command/Cargo.toml

```diff
@@ -90,6 +90,8 @@ toml = "0.7"
 unicode-segmentation = "1.10"
 ureq = { version = "2.7", default-features = false, features = ["charset", "gzip", "json", "native-tls"] }
 url = "2.2"
+uu_cp = { path = "../../../coreutils/src/uu/cp"}
+uucore = { path = "../../../coreutils/src/uucore"}
```

Comment:
What is this used for? I hoped we could do this without directly depending on anything but `uu_cp` (even though `uucore` would still be a transitive dependency).

Reply:
I tried to compile with and without uucore. This screenshot is with it, but also trying to reference the modes directly. 

![image](https://github.com/nushell/nushell/assets/343840/6c50c6ab-ee22-4be1-a459-989951b40fe8)